freqtrade/exchange/gateio.py

```diff
@@ -31,4 +32,18 @@ def validate_ordertypes(self, order_types: Dict) -> None:
 
         if any(v == 'market' for k, v in order_types.items()):
             raise OperationalException(
-                    f'Exchange {self.name} does not support market orders.')
+                f'Exchange {self.name} does not support market orders.')
+
+    def fetch_stoploss_order(self, order_id: str, pair: str, params={}) -> Dict:
+        return self.fetch_order(
+            order_id=order_id,
+            pair=pair,
+            params={'stop': True}
+        )
+
+    def cancel_stoploss_order(self, order_id: str, pair: str, params={}) -> Dict:
+        return self.cancel_order(
+            order_id=order_id,
+            pair=pair,
+            params={'stop': True}
```

Comment:
This does not work - the implementation in CCXT differs from what you're passing here (i assume it's missaligned in ccxt ...).

https://github.com/ccxt/ccxt/blob/a88cd68ffab7382b5767ff41c57724d3829a4d4e/js/gateio.js#L3115

Compared to fetch_order:

https://github.com/ccxt/ccxt/blob/a88cd68ffab7382b5767ff41c57724d3829a4d4e/js/gateio.js#L2933

So either, we have to use `isStop` here - or ccxt needs to align the cancel method to the fetch-method and use `stop`.

Reply:
https://github.com/ccxt/ccxt/pull/12329